WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/WatchDog.java

```diff
@@ -35,14 +43,18 @@
      */
     private boolean userProjectRegistrationCancelled = false;
 
+    private WindowFocusListener windowFocusListener;
+
 
     public WatchDog(Project project) {
-        WatchDog.project = project;
+        this.project = project;
     }
 
     public void initComponent() {
+        WatchDogUtils.setActiveProject(project);
         WatchDogGlobals.setLogDirectory(PluginManager.getPlugin(PluginId.findId("nl.tudelft.watchdog")).getPath().toString() + "/logs/");
```

Comment:
I would go with 
`PluginManager.getPlugin(PluginId.findId("nl.tudelft.watchdog")).getPath().toString() + File.separator + "logs" + File.separator);`
rather than 
`new File(PluginManager.getPlugin(PluginId.findId("nl.tudelft.watchdog")).getPath(), "logs").getAbsolutePath();`

Because I need separator at the end.